src/nvim/api/private/helpers.c

```diff
@@ -1619,14 +1619,20 @@ VirtText parse_virt_text(Array chunks, Error *err)
   return virt_text;
 }
 
-bool api_is_truthy(Object obj, const char *what, bool nil_truthy, Error *err)
+/// Force obj to bool.
+/// If it fails, returns false and sets err
+/// @param obj      The object to coerce to a boolean
+/// @param what     The name of the object, used for error message
+/// @param if_nil   What to return if the type is nil.
```

Comment:
`if_nil` sounds like "was the value nil". I like the old one better. or maybe `nil_value` (as boolean is a value in this context).